raiden/network/transport/matrix/utils.py

```diff
@@ -234,3 +236,71 @@ def validate_userid_signature(user: User) -> Optional[Address]:
     ):
         return None
     return address
+
+
+def sort_servers_closest(servers: Sequence[str]) -> Sequence[Tuple[str, float]]:
+    """Sorts a list of servers by http round-trip time
+
+    Params:
+        servers: sequence of http server urls
+    Returns:
+        sequence of pairs of url,rtt in seconds, sorted by rtt, excluding failed servers
+        (possibly empty)
+    """
+    if not {urlparse(url).scheme for url in servers} <= {'http', 'https'}:
```

Comment:
can we call `issubset` here instead of `<=`? It took me some time to understand this.

Reply:
Sure, they're equivalent, but will do for readability